packages/@glimmer/benchmark-env/src/benchmark/on-modifier.ts

```diff
@@ -13,11 +13,11 @@ interface OnModifierState {
 }
 
 class OnModifierManager implements InternalModifierManager<OnModifierState, object> {
```

Comment:
This is true, but means we need to refactor the benchmark-env a bit since the built-in on modifier needs to be imported. We should do this in a followup